commands/git/audit/gitaudit.go

```diff
@@ -0,0 +1,133 @@
+package audit
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	ioUtils "github.com/jfrog/jfrog-client-go/utils/io"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xsc/services"
+
+	sourceAudit "github.com/jfrog/jfrog-cli-security/commands/audit"
+	"github.com/jfrog/jfrog-cli-security/utils"
+	"github.com/jfrog/jfrog-cli-security/utils/gitutils"
+	"github.com/jfrog/jfrog-cli-security/utils/results"
+	"github.com/jfrog/jfrog-cli-security/utils/results/output"
+	"github.com/jfrog/jfrog-cli-security/utils/xsc"
+)
+
+type GitAuditCommand struct {
+	GitAuditParams
+	progress ioUtils.ProgressMgr
+}
+
+func NewGitAuditCommand() *GitAuditCommand {
+	return &GitAuditCommand{}
+}
+
+func (gaCmd *GitAuditCommand) CommandName() string {
+	return "git_audit"
+}
+
+func (gaCmd *GitAuditCommand) SetProgress(progress ioUtils.ProgressMgr) {
+	gaCmd.progress = progress
+}
+
+func (gaCmd *GitAuditCommand) ServerDetails() (*config.ServerDetails, error) {
+	return gaCmd.serverDetails, nil
+}
+
+func (gaCmd *GitAuditCommand) Run() (err error) {
+	if gaCmd.repositoryLocalPath, err = coreutils.GetWorkingDirectory(); err != nil {
+		return
+	}
+	// Detect git info
+	gitInfo, err := DetectGitInfo(gaCmd.repositoryLocalPath)
+	if err != nil {
+		return fmt.Errorf("failed to get git context: %v", err)
+	}
+	if gitInfo == nil {
+		// No Error but no git info = project is dirty
```

Comment:
I think the term "dirty" is a bit confusing.
I suggest we can use: working tree is dirty / you have uncommitted changes

Reply:
done